app/models/appointment.rb

```diff
@@ -103,22 +103,21 @@ def mark_patient_already_visited
   end
 
   def mark_patient_as_dead
-    self.patient.status = :dead
-    self.patient.save
+    patient.status = :dead
+    patient.save
   end
 
   def anonymized_data
-    { id: hash_uuid(id),
-      patient_id: hash_uuid(patient_id),
-      created_at: created_at,
-      registration_facility_name: facility.name,
-      user_id: hash_uuid(patient&.registration_user&.id),
-      scheduled_date: scheduled_date,
-      overdue: days_overdue > 0 ? 'Yes' : 'No',
-      status: status,
-      agreed_to_visit: agreed_to_visit,
-      remind_on: remind_on
-    }
+    {id: hash_uuid(id),
```

Comment:
This always takes some getting used to for me, but I understand the reasoning. You can read some of the discussion here: https://github.com/testdouble/standard/issues/26.

Totally open to trying this out for a couple weeks, I imagine the distinction will be worth it.